src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -268,7 +268,8 @@ function displayAjaxRetryMessageForPanelHeading($element) {
 }
 
 
-window.onload = function() {
+//Set on ready events
```

Comment:
Remove, quite a redundant comment don't you think? ;) I know you didn't put it but let's remove it for good